src/app/account/account-list-filter/account-list-filter.component.ts

```diff
@@ -39,11 +40,19 @@ export class AccountListFilterComponent implements OnInit {
   @Output()
   public groupingsChanged = new EventEmitter();
 
-  public stateTranslationToken(state): string {
-    return stateTranslations[state];
+  public get marginRight() {
+    return this.sidebarContainerService.isOpen.getValue()
+      ? this.sidebarContainerService.width.getValue()
+      : 0;
   }
 
+  constructor(private sidebarContainerService: SidebarContainerService) {}
+
   public ngOnInit() {
     this.groupings = reorderAvailableGroupings(this.groupings, this.selectedGroupings);
   }
+
+  public stateTranslationToken(state): string {
+    return stateTranslations[state];
```

Comment:
There are several getters for this. Maybe create a pipe for this?

Reply:
maybe we can do it in a ticket which connected with this components.
because regression extend testing